Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +11,25 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            if (array == null || array.Length == 0)
```

Comment:
length check doesn't matter, because a for loop will not run any times if it is less than 1